hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/EnrichedSpanConstants.java

```diff
@@ -18,6 +18,8 @@ public class EnrichedSpanConstants {
   public static final String GRPC_REQUEST_URL = "grpc.request.url";
   public static final String GRPC_REQUEST_ENDPOINT = "grpc.request.endpoint";
   public static final String DROP_TRACE_ATTRIBUTE = "drop.trace";
+  public static final String PEER_SERVICE_NAME = "PEER_SERVICE_NAME";
```

Comment:
we shouldnt keep it here if they are not being used here

Reply:
Keeping it here as it is required at multiple places